app/src/main/java/com/example/capstone/ProfileCreationFragment.java

```diff
@@ -22,12 +65,119 @@ public static ProfileCreationFragment newInstance(String param1, String param2)
+                Log.d(TAG, "Clicked");
+                onPickPhoto(v);
+            }
+        });
+
+        Glide.with(getActivity()).load(R.drawable.placeholder_image).into(ivProfilePicture);
+
+        btnCompleteProfile = getActivity().findViewById(R.id.btn_complete_profile);
+        btnCompleteProfile.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                Log.i(TAG, "onClick complete profile button");
+                String fullName = etFullName.getText().toString();
+                String school = etSchool.getText().toString();
+
+                if (fullName.matches("")) {
+                    Toast.makeText(getActivity(), "Please enter your full name.", Toast.LENGTH_SHORT).show();
+                } else {
+                    FirebaseUser user = mAuth.getCurrentUser();
+                    String firebase_uid = user.getUid();
+                    ParseQuery<User> query = ParseQuery.getQuery(User.class);
+                    query.whereEqualTo(User.KEY_FIREBASE_UID, firebase_uid);
+
+                    query.getFirstInBackground(new GetCallback<User>() {
+                        public void done(User user, ParseException e) {
+                            if (e == null) {
+                                user.put("full_name",fullName);
+                                if (!school.matches("")){
+                                    user.put("school", school);
+                                }
+                                if (profilePictureByteArr != null) {
+                                    ParseFile profilePicture = new ParseFile("profile_pic", profilePictureByteArr);
+                                    user.put("profile_picture", profilePicture);
+                                }
+                                user.saveInBackground(new SaveCallback() {
+                                    @Override
+                                    public void done(ParseException e) {
+                                        if (e == null) {
+                                            Toast.makeText(getActivity(), "Saved", Toast.LENGTH_SHORT).show();
+                                        } else {
+                                            Toast.makeText(getActivity(), "Failed to Save", Toast.LENGTH_SHORT).show();
+                                            Log.d(getClass().getSimpleName(), "User update error: " + e);
+                                        }
+                                    }
+                                });
+                            } else {
+                                e.printStackTrace();
+                            }
+                        }
+                    });
+
+                    Intent home = new Intent(getActivity(), HomescreenActivity.class);
+                    startActivity(home);
+                }
+            }
+        });
+    }
+
+    public void onPickPhoto(View view) {
```

Comment:
The `on` prefix is used for methods called by events. So `onClick()` is called by the click event performed by user.
But this method is not called by directly by an event, better name would be: `pickPhoto()`